apps/index.coffee

```diff
@@ -12,6 +12,11 @@ console.time('feed')
 app.use(require('../apps/feed'))
 console.timeEnd('feed')
 
+
+console.time('explore')
+app.use(require('../apps/explore'))
+console.timeEnd('explore')
```

Comment:
Why a separate app?

Reply:
Less confusing, it's not at all the same thing as explore and I always forget where it is.

Reply:
I'd much rather us start to merge apps where it makes sense. It takes time to boot and this way they share a package. So similar things should definitely be grouped.